resources/views/hidden-apps.php

```diff
@@ -0,0 +1,27 @@
+<?php
+$this->layout('layouts/plugin');
+?>
+
+<div>
+    <div>
+        <dt-copy-text value="<?php echo esc_url($magic_link); ?>"></dt-copy-text>
+    </div>
+
+    <dt-launcher-hidden-app-grid id="appGrid" app-data='<?php echo htmlspecialchars($data); ?>'
```

Comment:
@ancy2023 , let's change this component to be labeled" "dt-launcher-hidden-app-menu" for clarity.